cfme/automate/__init__.py

```diff
@@ -26,7 +26,7 @@ def in_customization(self):
 
     @property
     def is_displayed(self):
-        return self.in_customization and not self.buttons.is_dimmed
+        return self.in_customization and not self.buttons.is_dimmed and self.buttons.is_displayed
```

Comment:
`buttons.is_displayed` should be checked before `buttons.is_dimmed` - or even checked within the `is_dimmed` method.

Reply:
In `operator precedence` like `and` operation order not matter.